src/node_process_methods.cc

```diff
@@ -157,6 +157,24 @@ static void Kill(const FunctionCallbackInfo<Value>& args) {
   args.GetReturnValue().Set(err);
 }
 
+static void Rss(const FunctionCallbackInfo<Value>& args) {
+  Environment* env = Environment::GetCurrent(args);
+
+  size_t rss;
+  int err = uv_resident_set_memory(&rss);
+  if (err)
+    return env->ThrowUVException(err, "uv_resident_set_memory");
+
+  // Get the double array pointer from the Float64Array argument.
+  CHECK(args[0]->IsFloat64Array());
+  Local<Float64Array> array = args[0].As<Float64Array>();
+  CHECK_EQ(array->Length(), 1);
+  Local<ArrayBuffer> ab = array->Buffer();
+  double* fields = static_cast<double*>(ab->GetBackingStore()->Data());
+
+  fields[0] = rss;
```

Comment:
```suggestion
  args.GetReturnValue().Set(static_cast<double>(rss));
```
`Float64Array` is overkill if you're returning a single floating-point value. You can expose the method directly, without going through a JS method first.